homework-submission/sobhiweek2/src/App.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import './App.css';
+import todos from "./source/todos.json";
+import Header from "./source/Header";
+import Description from "./source/Description";
+import DeadLine from "./source/DeadLine";
+    
+
+export default class App extends React.Component {
+
+  state = { todos };
+
+  handelCheck = () => { this.setState({ todos: this.state.todos })};
```

Comment:
This should get a parameter and use it to update the todos list using the **map** function, Because it looks useless like this.